### PR TITLE
Changes So That Invite link dosent embed! 

### PR DESCRIPTION
**Pull request by ionadev**
### Changes Proposed in this Pull Request (List new items in CHANGELOG.MD)
This PR Just adds <> so that invite link dosent embed!
-

### Semver Classification

- [ ] This PR only includes documentation or non-code changes.
- [ ] This PR fixes a bug and does not change the (intended) framework interface.
- [ ] This PR adds methods or properties to the framework interface.
- [ ] This PR removes or renames methods or properties in the framework interface.

### CODE REVIEW

**Comment by kyranet** [comment]
Please use template literals to fix CI.

**Comment by ionadev** [comment]
means?

**Comment by ionadev** [comment]
can u fix that? im in clg ryt now

File: src/languages/en-US.js [hunk 1]
```diff
@@ -104,7 +104,7 @@ module.exports = class extends Language {
 			COMMAND_PINGPONG: (diff, ping) => `Pong! (Roundtrip took: ${diff}ms. Heartbeat: ${ping}ms.)`,
 			COMMAND_INVITE: () => [
 				`To add ${this.client.user.username} to your discord guild:`,
-				this.client.invite,
+				'<'+this.client.invite+'>',
```
Comment (gc) [inline src/languages/en-US.js]:
```js
`<${this.client.invite}>`,
```

**Comment by ionadev** [comment]
# Fixed! Used Template Literals!

**Pull request merged**